tensorflow/tools/ci_build/linux/rocm/run_gpu_multi.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ==============================================================================
+set -e
+set -x
+
+N_BUILD_JOBS=$(grep -c ^processor /proc/cpuinfo)
+
+echo ""
+echo "Bazel will use ${N_BUILD_JOBS} concurrent build job(s) and ${N_TEST_JOBS} concurrent test job(s)."
+echo ""
+
+# First positional argument (if any) specifies the ROCM_INSTALL_DIR
+ROCM_INSTALL_DIR=/opt/rocm-3.8.0
+if [[ -n $1 ]]; then
+    ROCM_INSTALL_DIR=$1
+fi
+
+# Run configure.
+export PYTHON_BIN_PATH=`which python3`
+
+export TF_NEED_ROCM=1
+export ROCM_PATH=$ROCM_INSTALL_DIR
+
+yes "" | $PYTHON_BIN_PATH configure.py
+
+# Run bazel test command. Double test timeouts to avoid flakes.
+bazel test \
+      --config=rocm \
+      -k \
+      --test_tag_filters=gpu \
```

Comment:
--test_tag_filters=multi_gpu ?

Reply:
i don't follow.  my understanding is that the test tag filters exclude tests with the specified tag(s).  so why would we exclude multi_gpu tests in run_gpu_multi.py?

Reply:
> test tag filters exclude tests with the specified tag(s)

incorrect.  they do both include and exclude

`--test_tag_filters=included_tag,-excluded_tag`

notice the `-` prefix for the `-excluded_tag`, and the lack of it in the `included_tag`

Reply:
https://docs.bazel.build/versions/master/command-line-reference.html#flag--test_tag_filters